Documentation Guide.md

```diff
@@ -58,7 +58,7 @@ Example (imagine a search or find method):
 
 **Better**
 
-`# @param entities [Sketchup::Entities] The entities context in which to search. Defaults to the current model's entities.`
+`@param entities [Sketchup::Entities] The entities context in which to search. Defaults to the current model's entities.`
```

Comment:
Oh, I only now noticed this was merged into the repo. We actually want `@param [Sketchup::Entities] entities` as that's how YARD will regenerate the stubs. So we follow that for least resistance. 

I still think I will be able to bulk reformat the tags in this PR by doing a second pass after integrating. I'mm make a note to adjust this comment as well.